google-cloud/Gemfile

```diff
@@ -2,37 +2,35 @@ source "https://rubygems.org"
 
 gemspec
 
-gem "rake", "~> 11.0"
-gem "google-cloud-core", path: "../google-cloud-core"
-gem "google-cloud-env", path: "../google-cloud-env"
+gem "gcloud-jsondoc",
+    git: "https://github.com/GoogleCloudPlatform/google-cloud-ruby.git",
+    branch: "gcloud-jsondoc"
 gem "google-cloud-bigquery", path: "../google-cloud-bigquery"
+gem "google-cloud-container", path: "../google-cloud-container"
```

Comment:
@geigerj It seems that we've missed adding some of the newer packages to this top-level Gemfile and in the meta-packages' gemspecs.